TLM/TLM/Resources/lang_pl.txt

```diff
@@ -231,4 +231,11 @@ Also_apply_to_left/right_turns_between_one-way_streets Uwzględnia również skr
 Scan_for_known_incompatible_mods_on_startup Uruchamiaj sprawdzanie przy starcie gry
 Ignore_disabled_mods Ignoruj wyłączone mody
 Traffic_Manager_detected_incompatible_mods Traffic Manager wykrył niekompatybilne mody
-Notify_me_if_there_is_an_unexpected_mod_conflict Powiadom mnie w razie nieoczekiwanej niezgodność modów
\ No newline at end of file
+Notify_me_if_there_is_an_unexpected_mod_conflict Powiadom mnie w razie nieoczekiwanej niezgodność modów
+Display_speed_limits_mph Display speed limits as MPH instead of km/h
+Miles_per_hour Miles per Hour
+Kilometers_per_hour Kilometers per Hour
+Road_signs_theme_mph Theme for MPH road signs
+theme_Square_US US signs
+theme_Round_UK British signs
+theme_Round_German German signs
```

Comment:
PL Translations to update and we can merge this branch then ;)
```
Display_speed_limits_mph Wyświetlaj limity prędkości w mph/h zamiast km/h
Miles_per_hour Mile na godzinę
Kilometers_per_hour Kilometry na godzinę
Road_signs_theme_mph Motyw dla znaków MPH 
theme_Square_US Stany zjednoczone
theme_Round_UK Wielka Brytania
theme_Round_German Niemcy
```

Reply:
Updated